matRad_fluenceOptimizationJO.m

```diff
@@ -364,7 +368,13 @@
         warning(['Optimizer ''' pln.propOpt.optimizer ''' not known! Fallback to IPOPT!']);
         optimizer = matRad_OptimizerIPOPT;
 end
-       
+
+if exist('loadwInit', 'var')
+    if loadwInit ==1
```

Comment:
Im not entirely convinced this 'loadwInit' is a great way to manage stored initial weights as it could end up being quite messy in practice. i would take it out